articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -110,21 +125,21 @@ var userAgentApplication = new Msal.UserAgentApplication(msalconfig.clientID, nu
 > |Where  |  |
 > |---------|---------|
 > |`ClientId`     |Application Id from the application registered in the Azure Portal|
-> |`authority`    |It is the authority URL. *null* sets the default authority to <i>https:<span/>//login.microsoftonline.com/common</i>|
-> |`loginCallBack`| Name of a callback method called after the authentication|
-> |`redirectUri`  |URL where users are sent after authentication against Azure AD v2 Endpoint|
+> |`authority`    |It is the authority URL. Passing *null* sets the default authority to <i>https:<span/>//login.microsoftonline.com/common</i>. If using a specific tenant, set authority to <i>https:<span/>//login.microsoftonline.com/&lt;tenant name or ID&gt;</i>|
```

Comment:
Suggestion: 
It is the authority URL. Passing *null* sets the default authority to `https://login.microsoftonline.com/common`. If your app is single-tenant (targeting accounts in one directory only), set this value to `https://login.microsoftonline.com/<tenant name or ID>`